deployment/hasura/metadata/databases/tables/sequencing/user_sequence.yaml

```diff
@@ -26,24 +29,24 @@ select_permissions:
 insert_permissions:
   - role: aerie_admin
     permission:
-      columns: [definition, seq_json, name, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
       check: {}
       set:
         owner: "x-hasura-user-id"
   - role: user
     permission:
-      columns: [definition,seq_json, name, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
       check: {}
       set:
         owner: "x-hasura-user-id"
 update_permissions:
   - role: aerie_admin
     permission:
-      columns: [definition,seq_json, name, owner, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
```

Comment:
I noticed we dropped the `owner` permission from the column was that on purpose?

Reply:
Emphasizing this, as `aerie_admin` should always be able to update `owner` fields

Reply:
I must have accidentally removed it, good catch.